apps/mobile/src/components/search/TokenCard/index.tsx

```diff
@@ -118,9 +116,17 @@ export const TokenCard: React.FC<LaunchCoinProps> = ({
             View token page
           </Button>
 
+          <Button onPress={() => {
```

Comment:
Add check if you are the owner to see this button.

Reply:
Alright no problem